aviary/subsystems/propulsion/motor/model/motor_mission.py

```diff
@@ -96,23 +97,34 @@ def setup(self):
         motor_group_max = om.Group()
 
         # these two groups are the same as those above
-        motor_group_max.add_subsystem('motor_map_max', MotorMap(num_nodes=n),
-                                      promotes_inputs=[(Dynamic.Mission.THROTTLE, 'max_throttle'),
-                                                       Aircraft.Engine.SCALE_FACTOR,
-                                                       Dynamic.Mission.RPM],
-                                      promotes_outputs=[(Dynamic.Mission.TORQUE, 'motor_max_torque'),
-                                                        'motor_efficiency'])
-
-        motor_group_max.add_subsystem('power_comp_max',
-                                      om.ExecComp('max_power = max_torque * pi * RPM / 30',
-                                                  max_power={'val': np.ones(
-                                                      n), 'units': 'kW'},
-                                                  max_torque={'val': np.ones(
-                                                      n), 'units': 'kN*m'},
-                                                  RPM={'val': np.ones(n), 'units': 'rpm'}),
-                                      promotes_inputs=[('max_torque', Aircraft.Engine.Motor.TORQUE_MAX),
-                                                       ('RPM', Dynamic.Mission.RPM)],
-                                      promotes_outputs=[('max_power', Dynamic.Mission.SHAFT_POWER_MAX)])
+        motor_group_max.add_subsystem(
+            'motor_map_max',
+            MotorMap(num_nodes=nn),
+            promotes_inputs=[
+                (Dynamic.Mission.THROTTLE, 'max_throttle'),
+                Aircraft.Engine.SCALE_FACTOR,
+                Dynamic.Mission.RPM,
+            ],
+            promotes_outputs=[
+                (Dynamic.Mission.TORQUE, 'motor_max_torque'),
+                'motor_efficiency',
+            ],
+        )
+
+        motor_group_max.add_subsystem(
+            'power_comp_max',
+            om.ExecComp(
+                'max_power = max_torque * pi * RPM / 30',
+                max_power={'val': np.ones(nn), 'units': 'kW'},
+                max_torque={'val': np.ones(nn), 'units': 'kN*m'},
+                RPM={'val': np.ones(nn), 'units': 'rpm'},
```

Comment:
is it possible to use `rad/s` as the units and let OM handle the unit conversion?

Reply:
Good idea, I'll try that out